pyproject.toml

```diff
@@ -28,7 +28,7 @@ localhost = [
 ]
 dev = [
```

Comment:
@mrakitin shall we describe the developer setup in the README under contribute? Such as we need to install dev dependencies: `pip install -e ."[dev]"`, and how to setup black-jupyter formatter.

Reply:
Good idea, @pranabdas!

Reply:
@pranabdas, I've added a step in the "Contribute" section explaining this: [753b669](https://github.com/Exabyte-io/api-examples/pull/80/commits/753b669eed1dff02ac399ad5e707d7961dd40a60). Please have a look.